### PR TITLE
HOTFIX: Emails not sending

### PR DESCRIPTION
**Pull request by TomekFraczek**
Email host settings were set to send emails to localhost not smtp

When porting settings to heroku, it seems we ended up with the dev
version of the email settings. Therefore, emails have simply been
printed to console instead of getting sent to the ionos smtp server to
be forwarded to the actual recipient. Updated settings to pull from the
env vars

### CODE REVIEW

File: excsystem/settings/prod2.py [hunk 1]
```diff
@@ -25,11 +25,14 @@
...
-MEMBERSHIP_EMAIL_HOST_PASSWORD = ""
-EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.ionos.com"
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+MEMBERSHIP_EMAIL_HOST_USER = os.environ.get("MEMBERSHIP_EMAIL_HOST_USER")
+MEMBERSHIP_EMAIL_HOST_PASSWORD = os.environ.get("MEMBERSHIP_EMAIL_HOST_PASSWORD")
+EMAIL_HOST_USER = MEMBERSHIP_EMAIL_HOST_USER
+EMAIL_HOST_PASSWORD = MEMBERSHIP_EMAIL_HOST_PASSWORD
```
Comment (estensen) [inline excsystem/settings/prod2.py]: Why not just set `EMAIL_HOST_USER` and `EMAIL_HOST_PASSWORD` as env vars directly?

**Comment by estensen** [comment]
@TomekFraczek After merging can you merge master into dev so master isn't ahead of dev?

File: excsystem/settings/prod2.py (hunk 1 above)
Comment (TomekFraczek) [reply excsystem/settings/prod2.py]: uhh.... legacy?

Good point though, will fix.

Comment (TomekFraczek) [reply excsystem/settings/prod2.py]: Added. Also added relevant env vars on heroku

**Pull request merged**